lib/cmr/base_client.rb

```diff
@@ -58,7 +59,7 @@ def token_header(token, use_real = false)
     end
 
     def request(method, url, params, body, headers)
-      Rails.logger.info "#{self.class} Request #{method} #{url} - Body: #{parse_string_for_tokens(body)} - Time: #{Time.now.to_s(:log_time)}"
+      # Rails.logger.info "#{self.class} Request #{method} #{url} - Body: #{parse_string_for_tokens(body)} - Time: #{Time.now.to_fs(:log_time)}"
```

Comment:
any reason we wanna comment this out? using to_fs makes sense here. I'd rather keep as many logs as we can since we don't have a great working understanding of the app across the team